sunpy/net/dataretriever/tests/test_vsm.py

```diff
@@ -0,0 +1,49 @@
+"""
+This module tests SOLIS VSM Client.
+"""
+#This module was developed with funding provided by
+#the Google Summer of Code 2016.
+import pytest
+
+from astropy import units as u
+from sunpy.net.vso.attrs import Time, Instrument, Physobs, Wavelength
+from sunpy.net.dataretriever.downloader_factory import UnifiedResponse
+from sunpy.net import Fido
+from sunpy.net import attrs as a
+import sunpy.net.dataretriever.sources.vsm as vsm
+
+VClient = vsm.VSMClient()
+
+trange = Time('2014/6/1', '2014/6/4')
+#Not downloading fits files since
+#they are very huge ranging from 75MB-100MB
+@pytest.mark.online
+@pytest.mark.parametrize("time, instrument, wavelength, physobs",
+[(trange, Instrument('vsm'), Wavelength(6302*u.AA),
+Physobs("LOS_MAGNETIC_FIELD"))])
+def test_query(time, instrument, physobs, wavelength):
+    qr = VClient.query(time, instrument, physobs, wavelength)
+    assert len(qr) == 3
+
+def test_can_handle_query():
+    assert VClient._can_handle_query(trange, Instrument('vsm'), Wavelength(6302*u.AA))
+    assert not VClient._can_handle_query(trange, Instrument('vsm'))
+    assert VClient._can_handle_query(trange, Instrument('vsm'), Wavelength(8542*u.AA))
+    assert VClient._can_handle_query(trange, Instrument('vsm'), Wavelength(1083.0*u.nm),
+                                     Physobs('EQUIVALENT_WIDTH'))
```

Comment:
```test query``` or ```test_can_handle_query```, Why would I parametrize either of them, they only have on test. Could you be more specific about what you want here ?